{{cookiecutter.project_slug}}/home/management/commands/load_initial_data.py

```diff
@@ -1,6 +1,7 @@
+from django.contrib.sites.models import Site
 from django.core.management import BaseCommand
-
-from home.models import CustomText, HomePage
+from home.models import CustomText
+from home.models import HomePage
 
```

Comment:
Did this need change?  Single line is fine.

Reply:
It was my editor, let me revert that